app/helpers/Formatters.scala

```diff
@@ -59,4 +73,30 @@ object Formatters {
     numberFormat.setMinimumFractionDigits(maxDecimalPlaces)
     numberFormat.format(amount.abs)
   }
+
+  def parseDateString(date: String): LocalDate = {
+    val dateArray: Array[String] = date.split(singleSpace)
+
+    val day = dateArray(0).toInt
+    val monthString = dateArray(1)
+    val year = dateArray(2).toInt
+
+    LocalDate.of(year, intValueForMonthString(monthString), day)
+  }
+
+  //scalastyle:off
+  private def intValueForMonthString(month: String) = month match {
```

Comment:
I know in some other repo's we have full month names in the English/Welsh translation files. Is it worth doing it that way (and thus keeping all the welsh in one file)?

Reply:
Has been discussed over the chat. It will be removed as part of refactoring of DailyStatementViewModel that will be done as part of separate ticket.